cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -84,6 +84,8 @@ type AutoscalingOptions struct {
 	CloudProviderName string
 	// NodeGroups is the list of node groups a.k.a autoscaling targets
 	NodeGroups []string
+	// RemoveUnregisterDelay sets the duration before the next remove unregister attempt
+	RemoveUnregisterDelay time.Duration
```

Comment:
Why not follow the existing naming scheme for options used for similar functionality here?

(e.g. something like `UnregisteredNodeRemovalDelayAfterAttempt`)